plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Repositories.js

```diff
@@ -231,6 +234,27 @@ Ext.define('NX.coreui.controller.Repositories', {
 
   /**
    * @private
+   * When creating a new repository, select the ‘default’ blobstore by default (if it exists)
+   */
+  selectDefaultBlobstore: function(combo) {
```

Comment:
Personally I think this should be a lot simpler - just select the first one in the list. 
And syntactically it should be as simple as:
```
combo.select(combo.getStore().getAt(0));
```